src/aliceVision/numeric/numeric.hpp

```diff
@@ -568,5 +568,12 @@ void SplitRange(const T range_start, const T range_end, const int nb_split,
   }
 }
 
+/**
+ * This function initializes the global state of random number generators that our current tests
+ * depend on. This makes it possible to have exactly reproducible tests without random failures.
+ * In order to introduce variation in test execution, ALICEVISION_TEST_RANDOM_SEED environment
+ * variable can be set to an integer value.
+ */
+void makeRandomOperationsReproducibleInTests();
```

Comment:
I would suggest to remove the "inTests" from the generic function name:
```suggestion
void makeRandomOperationsReproducible();
```
Even if it's only used in tests for now, it is not specific to unit tests in itself.

Same for the env var: `ALICEVISION_TEST_RANDOM_SEED` => `ALICEVISION_RANDOM_SEED`.

Reply:
Indeed, that's correct. Fixed now, thanks.